Lesson_10_Algorithm/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        int[] array = { 4, 9, 7, 6, 2, 3, 8, 5, 0, 1 };
+
+        SortMathod(array);
+
+        PrintArray(array);
+    }
+
+    private static void SortMathod(int[] array)
```

Comment:
Can you identify the used sort method? What pros & cons it has? What other types of sorting do you know?
Try to create a new method for implementing there another sorting algorithm. Check the execution time. The best option would be to compare the performance using benchmark

Reply:
| Method              | Mean      | Error    | StdDev   |
|-------------------- |----------:|---------:|---------:|
| BubbleSortBenchmark |  44.56 ns | 2.081 ns | 5.905 ns |
| QuickSortBenchmark  | 115.20 ns | 2.300 ns | 3.842 ns |